pkg/server/api/svid/v1/server_test.go

```diff
@@ -2,35 +2,88 @@ package svid_test
 
 import (
 	"context"
+	"crypto/rand"
+	"crypto/x509"
+	"net/url"
 	"testing"
+	"time"
 
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/spire/pkg/server/api"
+	"github.com/spiffe/spire/pkg/server/api/rpccontext"
 	"github.com/spiffe/spire/pkg/server/api/svid/v1"
 	svidpb "github.com/spiffe/spire/proto/spire-next/api/server/svid/v1"
 	"github.com/spiffe/spire/test/spiretest"
+	"github.com/spiffe/spire/test/testkey"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/codes"
 )
 
-func TestServer(t *testing.T) {
-	registerFn := func(s *grpc.Server) {
-		svid.RegisterService(s, FakeService{})
-	}
+func TestMintX509SVID(t *testing.T) {
+	log, _ := test.NewNullLogger()
+	ctx := rpccontext.WithLogger(context.Background(), log)
 
-	contextFn := func(ctx context.Context) context.Context {
-		return ctx
+	fakeService := &FakeService{}
+	client, done := createClient(ctx, t, fakeService)
+	defer done()
+
+	spiffeID := spiffeid.Must("example.org", "workload1")
+
+	// Create certificate request
+	key := testkey.NewEC256(t)
+	template := &x509.CertificateRequest{
+		SignatureAlgorithm: x509.ECDSAWithSHA256,
+		URIs:               []*url.URL{spiffeID.URL()},
 	}
+	csrRaw, err := x509.CreateCertificateRequest(rand.Reader, template, key)
+	require.NoError(t, err)
 
-	conn, done := spiretest.NewAPIServer(t, registerFn, contextFn)
-	defer done()
+	// Setup fake
+	expiresAt := time.Now()
+	fakeService.svid = &api.X509SVID{
+		ID:        spiffeID,
+		CertChain: []*x509.Certificate{},
```

Comment:
we should return something here so we can  verify the certificates are added appropriately to the response